### PR TITLE
Updated 0.35 release notes tables

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Updated the tables in the release notes.

[skip ci]

Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the release notes table with the document changes that have already been approved. I will update one more change that I am aware of after it is approved and merged. 

Please let me know if any changes are required or if any other issue has to be added or deleted in any of the tables.

The changes as per the review comments given in https://github.com/eclipse-openj9/openj9/pull/16088 will handled in that PR.

Thanks!

File: doc/release-notes/0.35/0.35.md [hunk 1]
```diff
@@ -49,45 +49,38 @@ The following table covers notable changes in v0.35.0. Further information about
 <tbody>
 
 <tr>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/15555">#15555</a></td>
+<td valign="top"> Java&trade; dump output contains a new `Initializing thread: <thread_name>` string in the existing `3XMTHREADBLOCK` line.</td>
```
Comment (pshipton) [inline doc/release-notes/0.35/0.35.md:53]: The back quotes aren't doing anything in the .md file. In other places `<tt>` is used instead. All the places that use back quotes need to be updated.

File: doc/release-notes/0.35/0.35.md [hunk 2]
```diff
@@ -49,45 +49,38 @@ The following table covers notable changes in v0.35.0. Further information about
...
+<td valign="top">All versions (All operating systems except Windows&reg;)</td>
+<td valign="top">On operating systems other than Windows&trade;, you can now use the `user2` event for the `-Xdump` option. The `user2` event is commonly used for taking system dump files with exclusive access without overriding the `user` event, which is generally left for taking Java dump files for performance investigations. </td>
 </tr>
 
 <tr>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
+<td valign="top"><a ref="https://github.com/eclipse-openj9/openj9/issues/14157">#14157</a></td>
```
[the remark this answers is not in the document]
Comment (Sreekala-Gopakumar) [reply doc/release-notes/0.35/0.35.md]: Corrected.

File: doc/release-notes/0.35/0.35.md (hunk 1 above)
Comment (Sreekala-Gopakumar) [reply doc/release-notes/0.35/0.35.md]: Corrected.

**Comment by Sreekala-Gopakumar** [comment]
Added the last issue that was documented after it was approved and merged (-XX:[+|-]JITServerLocalSyncCompiles enabled by default). Please check and confirm. Thanks!

File: doc/release-notes/0.35/0.35.md [hunk 3]
```diff
@@ -49,45 +49,45 @@ The following table covers notable changes in v0.35.0. Further information about
 <tbody>
 
 <tr>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/15555">#15555</a></td>
+<td valign="top"> Java&trade; dump output contains a new <tt>Initializing thread: <thread_name></tt> string in the existing <tt>3XMTHREADBLOCK</tt> line.</td>
```
Comment (pshipton) [inline doc/release-notes/0.35/0.35.md:53]: `<thread_name>` isn't showing up in the preview, I expect the `<`, `>` characters need to be handled specially.

Comment (Sreekala-Gopakumar) [reply doc/release-notes/0.35/0.35.md]: Corrected it.

**Review by pshipton — APPROVED** [review]

**Comment by pshipton** [comment]
@keithc-ca @mpirvu any concerns?

**Review by mpirvu — APPROVED** [review]
LGTM

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**